truss/templates/README.md.jinja

```diff
@@ -0,0 +1,5 @@
+{% if config.model_name == None %}
+## Model how-to
+{% else %}
+## {{ config.model_name }} how-to
+{% endif %}
```

Comment:
I know this is a WIP, and not sure what's coming but it would be good to put in code blocks for how to use the truss, eg 
- commands to run to build/run the docker container
- commands to run predict
- how to add a preprocessing/postprocessing step
-